action-whitelist/README.md

```diff
@@ -0,0 +1,70 @@
+# action-whitelist
+Scan `.github/workflows/` for `.yml files` and loop through all workflows, checking their versions against a list of approved authors and versions contained in `approved.json`.
+
+The output is stored with the name `actions`, which can be retrieved in another action with `${{ steps.<step id>.outputs.actions }}`.
+
+Used in lieu of Github Enterprise [Allow specified actions](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-specific-actions-to-run) for private repositories.
+
+## Example usage
+Minimal uses expression to use this action:
+
+``` yaml
+uses: ./actions-whitelist
```

Comment:
Let's change the name to `actions-allowedlist` or something, to avoid the connotations of "whitelist" and (implied) "blacklist".

Reply:
changed the action name everywhere including the directory